app/controllers/application_controller.rb

```diff
@@ -93,7 +93,9 @@ def check_browser
     return if request_format.xml? || request_format.json?
 
     if current_user && browser_not_modern?
-      logout_keeping_session!
+      logout_killing_session!
+      destroy_user_session!
```

Comment:
I see the content is this:
https://github.com/3scale/porta/blob/1506f97e3023964551c28c4a37d45481dd98b59a/app/lib/authenticated_system.rb#L66-L71
However, this line is not necessary because it is already called from `reset_session`:
https://github.com/3scale/porta/blob/1506f97e3023964551c28c4a37d45481dd98b59a/app/lib/authenticated_system.rb#L20-L24
which is called from the previous line inside `logout_killing_session!`:
https://github.com/3scale/porta/blob/1506f97e3023964551c28c4a37d45481dd98b59a/app/lib/authenticated_system.rb#L197